### PR TITLE
IO documentation

### PR DESCRIPTION
**Pull request by bjuncek**
Adding for clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,17 @@ Since each time the script is run the list of input files is shuffled, you don't
 workers will be processing the same video.
 On a rare occasion when the collision happens, the script will rewrite previously extracted features.
 
+
+## Input
+The inputs are paths to video files that are readable by OpenCV VideoReader.
```
Comment (v-iashin) [inline README.md:88]: Actually, not all of them are readable by OpenCV VideoReader.

r21d and s3d rely on torchvision.io.read_video

File: README.md [hunk 2]
```diff
@@ -83,6 +83,17 @@ Since each time the script is run the list of input files is shuffled, you don't
...
+Paths can be passed as a list of paths or as a text file formatted with a single path per line.
+
+
+## Output
+Output is defined by the `on_extraction` argumenty; by default it prints the featrues to the command line.
```
Comment (v-iashin) [inline README.md:93]: "argumenty" and "featrues" -- could you patch these? otherwise, no worries, I will do it myself.

File: README.md [hunk 3]
```diff
@@ -83,6 +83,17 @@ Since each time the script is run the list of input files is shuffled, you don't
...
+Possible values of output are ['print', 'save_numpy', 'save_pickle']. `save_*` options save the features in
+the `ouptut_path` folder with the same name as the input video file but with the `.npy` or `.pkl` extension.
```
Comment (v-iashin) [inline README.md:95]: `ouptut_path` :)

File: README.md (hunk 1 above)
Comment (bjuncek) [reply README.md:88]: Or just removing it is fine, I'll patch it. The whole purpose of this PR is to have a description of what goes in and what goes out :)

**Comment by bjuncek** [comment]
Note to self: my spellchecker extension is not working. 
Patched I think

**Comment by v-iashin** [comment]
no need to bother. life is too short to worry about these kinds of things anyway..

**Pull request merged**